docs/updating/update-to-43.md

```diff
@@ -16,8 +16,111 @@ modified_at: 2024-07-31
 
 ## Update to CKEditor&nbsp;5 v43.0.0
 
-_Released on August xx, 2024._
+_Released on August 7, 2024._
 
 For the entire list of changes introduced in version 43.0.0, see the [release notes for CKEditor&nbsp;5 v43.0.0](https://github.com/ckeditor/ckeditor5/releases/tag/v43.0.0).
 
-Below are the most important changes that require your attention when upgrading to CKEditor&nbsp;5 v43.0.0.
\ No newline at end of file
+Below are the most important changes that require your attention when upgrading to CKEditor&nbsp;5 v43.0.0.
+
+### Export to Word v2 becomes the default
+
+In version v42.0.0, we introduced new version of Export to Word plugin that utulizes updated v2 of the converter. It was introduced in an opt-in manner. From the v43.0.0, the Export to Word v2 is **the default** configuration. This may require updates to the editor's configuration if you use this plugin.
+
+Example v1 configuration:
+
+```js
+exportWord: {
+            tokenUrl: 'https://example.com/cs-token-endpoint',
+            fileName: 'my-file.docx',
+            converterOptions: {
+                format: 'A4',
+                margin_top: '20mm',
+                margin_bottom: '20mm',
+                margin_right: '12mm',
+                margin_left: '12mm'
+            }
+        }
+```
+
+Migrated to v2:
+
+```js
+exportWord: {
+            tokenUrl: 'https://example.com/cs-token-endpoint',
+            fileName: 'my-file.docx',
+            converterOptions: {
+                document: {
+                    size: 'A4',
+                    margin: {
+                        top: '20mm',
+                        bottom: '20mm',
+                        right: '12mm',
+                        left: '12mm'
+                    }
+                }
+            }
+        }
+```
+
+The full v2 configuration can be found in the {@link module:export-word/exportword~ExportWordConverterOptionsV2} API documentation.
+
+Alongside with the configuration changes we decided to switch the default `auto_pagination` option to `false`. This option was only used along our `Pagination` feature. When this option is enabled the breaks in the pagination force the breaks in the exported Word document. Browser engines and Microsoft Word differ significantly. Because of that we decided to turn this option to `false` If you still want to enforce breaks from pagination, change it in the configuration to `true`.
```

Comment:
```suggestion
Alongside the configuration changes, we switched the default `auto_pagination` option to `false`. We use the new default along with our `Pagination` feature. When this option is enabled, the breaks in the pagination force the breaks in the exported Word document. Browser engines and Microsoft Word differ significantly. Because of that, we decided to turn this option to `false`. If you still want to enforce breaks from pagination, change it in the configuration to `true`.
```